BibBuddy/app/src/main/java/de/bibbuddy/SearchModel.java

```diff
@@ -37,35 +35,52 @@ public SearchModel(Context context) {
    *
    * @return Returns the searchResultList
    */
-  public List<SearchItem> getSearchResultList(String searchInput, SearchSortCriteria sortCriteria) {
+  public List<SearchItem> getSearchResultList(String searchInput, SearchSortCriteria sortCriteria,
+                                              boolean[] filterCriteria) {
     searchResultList.clear();
 
-    // TODO filter
+    if (filterCriteria[0]) { // filter shelf
```

Comment:
Since there is one instruction inside the if-statement, brackets are not needed. This is also recommended by Android Studio.

Reply:
It is bad practice to use one line if statements without braces because if you accidently add a line it produces errors and it does not improve the readability. And one line if statements were not used in our code before.